<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionarNegos</class>
 <widget class="QDialog" name="gestionarNegos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="editNegos">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>405</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelOrigen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Origen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOrigen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="placeholderText">
     <string>Origen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDestino">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>191</red>
          <green>191</green>
          <blue>191</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="placeholderText">
     <string>Destino</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDestino">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Destino&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFecha">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fecha vuelo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGuion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;-&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFechaDis">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fecha dis.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlazas">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Plazas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="plazas">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="porcentaje">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelPorcDis">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;% Dis.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateVuelo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>38</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeVuelo">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateDis">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>38</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModificar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAcpetar">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBorrar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QFrame" name="filtrado">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboOrigenFilter">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>7</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFechaFilter">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fecha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOwnerFilter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Owner&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDestinoFilter">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Destino&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboOwnerFilter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>7</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateFechaFilter">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigenFilter">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Origen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboDestinoFilter">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>7</y>
      <width>135</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkFecha">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tablaNegos">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>800</width>
     <height>320</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboOwnerFilter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>gestionarNegos</receiver>
   <slot>selectedOwner()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboOrigenFilter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>gestionarNegos</receiver>
   <slot>selectedOrigen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboDestinoFilter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>gestionarNegos</receiver>
   <slot>selectedDestino()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateFechaFilter</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>gestionarNegos</receiver>
   <slot>selectedFecha()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tablaNegos</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>gestionarNegos</receiver>
   <slot>showLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFecha</sender>
   <signal>clicked(bool)</signal>
   <receiver>gestionarNegos</receiver>
   <slot>selectedFecha()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNuevo</sender>
   <signal>clicked()</signal>
   <receiver>gestionarNegos</receiver>
   <slot>nuevo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnModificar</sender>
   <signal>clicked()</signal>
   <receiver>gestionarNegos</receiver>
   <slot>modificar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBorrar</sender>
   <signal>clicked()</signal>
   <receiver>gestionarNegos</receiver>
   <slot>borrar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAcpetar</sender>
   <signal>clicked()</signal>
   <receiver>gestionarNegos</receiver>
   <slot>aceptar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>gestionarNegos</receiver>
   <slot>cancelar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>802</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectedOwner()</slot>
  <slot>selectedOrigen()</slot>
  <slot>selectedDestino()</slot>
  <slot>selectedFecha()</slot>
  <slot>showLabel()</slot>
  <slot>activarFecha()</slot>
  <slot>nuevo()</slot>
  <slot>modificar()</slot>
  <slot>borrar()</slot>
  <slot>aceptar()</slot>
  <slot>cancelar()</slot>
 </slots>
</ui>
